protocols/v2/framing-sv2/src/framing2.rs

```diff
@@ -64,11 +70,12 @@ pub trait Frame<'a, T: Serialize + GetSize>: Sized {
     ) -> Option<Self>;
 }
 
+/// Abstraction for a SV2 Frame.
```

Comment:
I would add some info about the Sv2Frame in general.

Reply:
Ignore the  general info about the Sv2Frame in general I didn't see that you have a module level comment for that.

Reply:
@Fi3 can you elaborate on the comments here? I'm confused about the suggestion

Reply:
I was suggesting to add thing that you already added so just ignore it